.ci/ironbank-validation.groovy

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'ubuntu-20 && immutable' }
+  environment {
+    REPO = 'beats'
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    PIPELINE_LOG_LEVEL = "INFO"
+    BEATS_FOLDER = "x-pack/heartbeat"
+    SLACK_CHANNEL = '#beats'
+    NOTIFY_TO = 'observability-robots-internal+ironbank-beats-validation@elastic.co'
+  }
+  options {
+    timeout(time: 1, unit: 'HOURS')
+    buildDiscarder(logRotator(numToKeepStr: '20', artifactNumToKeepStr: '20', daysToKeepStr: '30'))
+    timestamps()
+    ansiColor('xterm')
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+    disableConcurrentBuilds()
+  }
+  stages {
+    stage('Checkout') {
+      options { skipDefaultCheckout() }
+      steps {
+        deleteDir()
+        gitCheckout(basedir: "${BASE_DIR}")
+        setEnvVar("GO_VERSION", readFile("${BASE_DIR}/.go-version").trim())
+        dir("${BASE_DIR}"){
+          setEnvVar('BEAT_VERSION', sh(label: 'Get beat version', script: 'make get-version', returnStdout: true)?.trim())
+        }
+      }
+    }
+    stage('Package'){
+      options { skipDefaultCheckout() }
+      steps {
+        withMageEnv(){
+          dir("${env.BASE_DIR}/${env.BEATS_FOLDER}") {
+            sh(label: 'make ironbank-package', script: "make -C ironbank package")
+          }
+        }
+      }
+      post {
+        failure {
+          notifyStatus(slackStatus: 'danger', subject: "[${env.REPO}@${BRANCH_NAME}] package for ${env.BEATS_FOLDER}", body: "Contact the heartbeats team. (<${env.RUN_DISPLAY_URL}|Open>)")
+        }
+      }
+    }
+    stage('Ironbank'){
+      options { skipDefaultCheckout() }
+      steps {
+        withMageEnv(){
+          dir("${env.BASE_DIR}/${env.BEATS_FOLDER}") {
+            sh(label: 'mage ironbank', script: 'mage ironbank')
+          }
+        }
+      }
+      post {
+        failure {
+          notifyStatus(slackStatus: 'danger', subject: "[${env.REPO}@${BRANCH_NAME}] Ironbank docker context for ${env.BEATS_FOLDER}", body: "Contact the @observablt-robots-team team. (<${env.RUN_DISPLAY_URL}|Open>)")
+        }
+      }
+    }
+    stage('Validate'){
+      options { skipDefaultCheckout() }
+      steps {
+        withMageEnv(){
+          dir("${env.BASE_DIR}/${env.BEATS_FOLDER}") {
+            sh(label: 'make validate-ironbank', script: "make -C ironbank validate-ironbank")
+          }
+        }
+      }
+      post {
+        failure {
+          notifyStatus(slackStatus: 'danger', subject: "[${env.REPO}@${BRANCH_NAME}] Ironbank validation failed", body: "Contact the @observablt-robots-team team. (<${env.RUN_DISPLAY_URL}|Open>)")
+        }
```

Comment:
This is the new validation, so if the docker image cannot be built then something is broken in the list of dependencies, if that happens then we will need to fix it.